### PR TITLE
:robot: Package update

### PR DESCRIPTION
**Pull request by vm-packages**
Automated package update

### CODE REVIEW

**Review by mr-tz — APPROVED** [review]

File: packages/exiftool.vm/exiftool.vm.nuspec [hunk 1]
```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>exiftool.vm</id>
-    <version>12.55</version>
+    <version>12.55.0</version>
```
Comment (MalwareMechanic) [inline packages/exiftool.vm/exiftool.vm.nuspec:5]: This should be `12.55.0.20230121`

**Review by MalwareMechanic — CHANGES_REQUESTED** [review]

File: packages/exiftool.vm/exiftool.vm.nuspec (hunk 1 above)
Comment (mr-tz) [reply packages/exiftool.vm/exiftool.vm.nuspec:5]: 2023-01-26_16-14-08_firefox

Comment (Ana06) [reply packages/exiftool.vm/exiftool.vm.nuspec:5]: I am not sure we should copy the date from the dependency. If we make a modification in the package we would need to modify the date as well. Our script only adds the date if the version is the same as before the update (when it is strictly necessary): https://github.com/mandiant/VM-Packages/blob/main/scripts/utils/update_package.py#L11

Comment (MalwareMechanic) [reply packages/exiftool.vm/exiftool.vm.nuspec:5]: Ah, you're right we should just add the current date

Comment (mr-tz) [reply packages/exiftool.vm/exiftool.vm.nuspec:5]: I have a fix for this. Also looking at the other version issue (#212).

**Review by mr-tz — APPROVED** [review]
lgtm, #223 should be merged first

**Pull request merged**